src/styles/_view_lineup.scss

```diff
@@ -302,12 +302,20 @@ $lu_assets: '~lineupjs/src/assets';
         bottom: 0;
         right: 0;
         left: 0;
-        overflow-y: auto; // make tab pane scrollable for long content
-      }
+        flex-direction: column;
```

Comment:
Could we move this into the `&.active` block, so that `display: flex` belongs together?